### ISSUE TITLE
Hmm. Is there more than just this?

### ISSUE THREAD

**Issue by neilyoung** [mariogasparoni/kurento-monitor#6]
[16-06 13:41:23] 1
[16-06 13:41:24] 1
[16-06 13:41:25] 1
[16-06 13:41:26] 1
[16-06 13:41:27] 1
[16-06 13:41:28] 1
[16-06 13:41:29] 1
[16-06 13:41:30] 1
[16-06 13:41:31] 1
[16-06 13:41:32] 1
[16-06 13:41:33] 1
[16-06 13:41:34] 0
[16-06 13:41:35] 0
[16-06 13:41:36] 0
[16-06 13:41:37] 0

So well, if not then...

**Comment by mariogasparoni** [comment]
Yes, if you set `"pipelines_only": 0`  in our default.json ,you should be able to see more info (and not only the number of pipelines)
